R/SouthAfrica.R

```diff
@@ -0,0 +1,85 @@
+#' SouthAfrica Class for downloading, cleaning and processing notification data
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for South Africa.
+#'
+#' @details Inherits from `DataClass`
+#' @source https://raw.githubusercontent.com/dsfsi/covid19za/master/data/covid19za_provincial_cumulative_timeline_confirmed.csv # nolint
+#' @source https://raw.githubusercontent.com/dsfsi/covid19za/master/data/covid19za_provincial_cumulative_timeline_deaths.csv # nolint
```

Comment:
`nolint` doesn't work in Rd source, put `# nolint start` and `# nolint end` around these lines